### DISCUSSION TITLE
Delete Account

### DISCUSSION THREAD

**Discussion by Austin-HTTPS** [mastodon/mastodon#28544] [Q&A]
I have an account on mastodon.social and would like to delete my account, how do I go by doing it?

**Comment by StrangeMinds1960** [comment]
If you're using the web....preferences > account > delete account

**Reply by Austin-HTTPS** [reply]
I don't see the account thing.
![image_2024-01-01_140230798](https://github.com/mastodon/mastodon/assets/65206545/a65a853b-8720-451c-bac5-404bee678825)

**Comment by StrangeMinds1960** [comment]
It's right there on your screenshot.  The 'Preferences' I meant was the cog icon above where you create a post (if you hover over it it's called Preferences)

**Reply by Austin-HTTPS** [reply]
I see. Thank you so very much!
